### DISCUSSION TITLE
Command line doesn't work

### DISCUSSION THREAD

**Discussion by dhythm** [vscode-neovim/vscode-neovim#1526] [Q&A] [answered]
The following issue is closed but it happens for me after updating vscode-neovim.
Does anyone have a same issue or the solution of this?

```
❯ nvim -v
NVIM v0.9.0
```

<img width="303" alt="image" src="https://github.com/vscode-neovim/vscode-neovim/assets/10891318/2a05dbb7-e952-44ae-94d8-c5e35c4a7f95">

https://github.com/vscode-neovim/vscode-neovim/issues/1298

**Comment by xiyaowong** [comment]
#1508 breaks this

**Comment by xiyaowong** [comment, marked as answer]
fixed by #1533

**Reply by dhythm** [reply]
Sorry for late reply and thank for telling me!
